File: app/src/main/java/ua/regin/wallpapers/entity/Image.java
package ua.regin.wallpapers.entity;

import java.io.Serializable;

public class Image implements Serializable{

    private String url;
    private PreviewImage thumb;
    private PreviewImage preview;

    public String getUrl() {
        return url;
    }

    public PreviewImage getThumb() {
        return thumb;
    }

    public PreviewImage getPreview() {
        return preview;
    }
}

File: app/src/main/java/ua/regin/wallpapers/entity/Image.kt
package ua.regin.wallpapers.entity;

import java.io.Serializable;

class Image : Serializable {

    var url: String;
    var thumb: PreviewImage;
    var preview: PreviewImage;

    constructor(url: String, thumb: PreviewImage, preview: PreviewImage) {
        this.url = url
        this.thumb = thumb
        this.preview = preview
    }
}
